docs/start.rst

```diff
@@ -63,8 +65,8 @@ Compile the program
 
 This will generate the output files in *test_output* directory.
 
-A good place to get started with with your own programs might be to copy `examples/getting_started.py` to some folder of your choice and start modiifying it. For further examples, have a look at the test programs inside the "tests" directory.
+A good place to get started with with your own programs might be to copy `examples/getting_started.py` to some folder of your choice and start modifying it. For further examples, have a look at the test programs inside the "tests" directory.
```

Comment:
We could include examples in the docs directly, Anton Ahkmerov (TUD professor) has made a tool to directly include code (with output ) in sphinx documentation. 

I think this is out of scope for now but definitely worth looking into at some point. 
https://blog.jupyter.org/integrating-output-in-documentation-with-jupyter-sphinx-ecf569ddab85